gcloud-java-examples/src/main/java/com/google/gcloud/examples/StorageExample.java

```diff
@@ -121,7 +104,7 @@ public String params() {
       }
       Blob[] blobs = new Blob[args.length - 1];
       for (int i = 1; i < args.length; i++) {
-        blobs[i - 1] = new Blob(storage, args[0], args[i]);
+        blobs[i - 1] = Blob.load(storage, args[0], args[i]);
```

Comment:
Here and in other places in this example I don't think we handle well a `null` `Blob`.

Maybe the simplest way to deal with that is to make the `parse` methods return 
`BlobId` rather than `Blob`.
Which maybe makes sense as in some cases we don't need the `Blob`.